plugins/meta/firewall/README.md

```diff
@@ -133,3 +133,17 @@ of the container as shown:
 - `-s 10.88.0.2 -m conntrack --ctstate RELATED,ESTABLISHED -j CNI-FORWARD`
 - `-d 10.88.0.2 -j CNI-FORWARD`
 
+A new chain, CNI-ADMIN is added to the CNI-FORWARD chain. CNI-ADMIN is intended as a user-controlled chain that all traffic passes through prior to the rules managed by the `firewall` plugin.
+
+`CNI-FORWARD` chain:
```

Comment:
I dont' think this section is required, becuase the forward chain doesn't acutally jump to the admin chain. Simply that when a packet hits the FILTER table, it gets sent to CNI-ADMIN for evaluation before being sent to CNI-FORWARD.

Reply:
hey @dcbw  does the comment make sense with the changes ?

Reply:
Actually looking at the code @dcbw's statement "the forward chain doesn't actually jump to the admin chain" seems to be false.

Reply:
Hey @bboreham 
What do you suggest ?
IMHO documenting that the CNI-ADMIN chain is available to utilize by users is the most valuable

Reply:
@xtreme-sameer-vohra I still think my comment applies but I should word it better... 

1) a chain is not added to any other chain, they jump between each other. Semantics mostly but let's be correct here
2) CNI-ADMIN and FORWARD chain do not directly interact. Instead, FORWARD chain jumps to CNI-FORWARD chain which in turn jumps to CNI-ADMIN

Reply:
How about this wording instead:

The CNI-FORWARD chain first sends all traffic to the CNI-ADMIN chain, which is intended as a user-controlled chain for custom rules that run prior to the rules managed by the `firewall` plugin. The `firewall` plugin does not add, delete, or modify rules in the CNI-ADMIN chain.

Reply:
> How about this wording instead:
> 
> The CNI-FORWARD chain first sends all traffic to the CNI-ADMIN chain, which is intended as a user-controlled chain for custom rules that run prior to the rules managed by the `firewall` plugin. The `firewall` plugin does not add, delete, or modify rules in the CNI-ADMIN chain.

Thanks, updated the PR.